### ISSUE TITLE
Python wheels

### ISSUE THREAD

**Issue by malmeloo** [Dadoum/pyprovision#10]
I've been looking into using pyprovision for FindMy.py for a little while now, and this morning I managed to build a Python wheel using [repairwheel](https://pypi.org/project/repairwheel/) that bundles external dependencies such as `libphobos2`. This allows people to install and use pyprovision without needing a D compiler.

I was wondering if you'd be interested in building such Python wheels as part of the pyprovision project, and potentially publishing the project to PyPi? I'd be happy to help with setting up an Actions workflow or something similar.

**Comment by maelp** [comment]
Would be interested in this too

**Comment by malmeloo** [comment]
I built [Anisette.py](https://github.com/malmeloo/anisette.py) in the meantime to generate Anisette headers, which depends on unicorn instead. Unicorn does have proper Python wheels and is easily buildable if they do happen to be missing, so it might suit your use case as well.

**Comment by maelp** [comment]
@malmeloo nice! So this completely replaces the Anisette server right?

**Comment by malmeloo** [comment]
Yep! It's already implemented in FindMy.py, but you can also use the library standalone if you wish.

**Comment by maelp** [comment]
@malmeloo very nice thanks!

I see this requires downloading some binaries, are those built for every platform? is it possible to recompile them if I need?

**Comment by malmeloo** [comment]
The binaries just contain the arm libraries found in the Apple music APK. They're platform agnostic; Anisette.py runs a VM (using unicorn/qemu) that runs the libraries. And unicorn comes with Python wheels, so pip will figure it out and you generally don't need to compile anything yourself.

**Comment by maelp** [comment]
Very clear, thanks!

**Comment by Dadoum** [comment]
I should probably archive the repo and add a link to Anisette.py in the readme. 

The emulator route is less of a hassle in Python and works consistently on every computer.
